sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestDocumentation.java

```diff
@@ -77,10 +76,9 @@ public void testFiveMinuteSqlObjectExample() throws Exception {
         });
     }
 
-
     @Test
     public void testObtainHandleViaOpen() throws Exception {
-        try (Handle handle = dbRule.getJdbi().open()) {}
+        assertThatCode(dbRule.getJdbi()::open).doesNotThrowAnyException();
```

Comment:
Leaks connection.

Reply:
True, but doesn't the process end and kill the db anyway? Writing a fix anyway, just interested in the mechanics behind memdbs in tests

Reply:
Isn't the sharedHandle in H2DatabaseRule et al also a leak then btw?

Reply:
The test rule closes the connection in its `after()` method.

Reply:
Well it closes the connection it gets from the handle. Closing the handle itself isn't important then?